### ISSUE TITLE
Assign role/remove tab order from map credits & esri logo

### ISSUE THREAD

**Issue by allisonmuise** [Esri/crowdsource-reporter#399]
Should they be treated like links?

![11-14-2018 3-07-26 pm](https://user-images.githubusercontent.com/2752195/48518769-7f2bb080-e81f-11e8-8749-a618f212e65a.png)
![11-14-2018 3-07-11 pm](https://user-images.githubusercontent.com/2752195/48518771-7f2bb080-e81f-11e8-86fa-83caab65194a.png)

**Comment by ashishchoure** [comment]
@allisonmuise 

We have added `aria-expanded` attribute to attribution widget, so we think the role should be "button". For esri logo, as it works like a link and opens the [esri.com](https://www.esri.com/en-us/home) website, the role should be "link".

**Comment by allisonmuise** [comment]
@ashishchoure sounds good to me

**Comment by allisonmuise** [comment]
verified
